### PR TITLE
Peaq network v0.9.29

### PR DESCRIPTION
**Pull request by sfffaaa**
Upgrade the apps to 0.9.29.

### CODE REVIEW

**Comment by niazhussain** [comment]
:star: PR built and available in a preview **peaqnetwork-apps-pr-6** [here](http://apps-pr6.ci.peaq.network)

**Comment by niazhussain** [comment]
:star: PR built and available in a preview **peaqnetwork-apps-pr-6** [here](http://apps-pr6.ci.peaq.network)

**Comment by niazhussain** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **dorloechter**
You can assign the PR to them by writing `/assign @dorloechter` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://jenkins-x.io/v3/develop/reference/chatops/?repo=peaqnetwork%2Fapps).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/peaqnetwork/apps/blob/dev/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by niazhussain** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **dorloechter**
You can assign the PR to them by writing `/assign @dorloechter` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://jenkins-x.io/v3/develop/reference/chatops/?repo=peaqnetwork%2Fapps).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/peaqnetwork/apps/blob/dev/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: packages/apps-config/src/api/spec/subsocial.ts [hunk 1]
```diff
@@ -1,9 +1,12 @@
 // Copyright 2017-2022 @polkadot/apps-config authors & contributors
 // SPDX-License-Identifier: Apache-2.0
 
+// IMPORTANT
+//
 // We only load the definitions here explicitly - if we try to go via
-//   import { types } from '@subsocial/types/substrate';
-// we end up with multiple version of types/API
-import definitions from '@subsocial/types/substrate/interfaces/subsocial/definitions';
+//   import { types } from '@subsocial/types';
```
Comment (kamrangul2404) [inline packages/apps-config/src/api/spec/subsocial.ts:7]: if import is not required , please remove it

File: packages/react-components/src/AccountIndex.tsx [hunk 2]
```diff
@@ -32,7 +32,7 @@ function AccountIndex ({ children, className = '', defaultValue, label, value }:
     [info]
   );
 
-  if (!api.query.indices) {
+  if (!api || !api.query.indices) {
```
Comment (kamrangul2404) [inline packages/react-components/src/AccountIndex.tsx]: can we replace `!api || !api.query.indices` with `!api?.query.indices`

File: packages/apps-config/src/api/spec/subspace.ts [hunk 3]
```diff
@@ -35,9 +36,14 @@ function extractAuthor (
   const preRuntimes = digest.logs.filter(
     (log) => log.isPreRuntime && log.asPreRuntime[0].toString() === 'SUB_'
   );
+
+  if (!preRuntimes || preRuntimes.length === 0) {
```
Comment (kamrangul2404) [inline packages/apps-config/src/api/spec/subspace.ts]: `preRuntimes.length === 0` can be replaced with `!preRuntimes.length `

**Review by kamrangul2404 — CHANGES_REQUESTED** [review]
@sfffaaa kindly perform the changes if there is no impact

**Comment by niazhussain** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **dorloechter**
You can assign the PR to them by writing `/assign @dorloechter` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://jenkins-x.io/v3/develop/reference/chatops/?repo=peaqnetwork%2Fapps).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/peaqnetwork/apps/blob/dev/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by niazhussain** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **dorloechter**
You can assign the PR to them by writing `/assign @dorloechter` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://jenkins-x.io/v3/develop/reference/chatops/?repo=peaqnetwork%2Fapps).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/peaqnetwork/apps/blob/dev/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: packages/apps-config/src/api/spec/subsocial.ts (hunk 1 above)
Comment (sfffaaa) [reply packages/apps-config/src/api/spec/subsocial.ts:7]: About this, I need more confidence to remove it. So, I'd like to keep this change.

File: packages/apps-config/src/api/spec/subspace.ts (hunk 3 above)
Comment (sfffaaa) [reply packages/apps-config/src/api/spec/subspace.ts]: done

File: packages/react-components/src/AccountIndex.tsx (hunk 2 above)
Comment (sfffaaa) [reply packages/react-components/src/AccountIndex.tsx]: done

**Comment by niazhussain** [comment]
:star: PR built and available in a preview **peaqnetwork-apps-pr-6** [here](http://apps-pr6.ci.peaq.network)

**Comment by niazhussain** [comment]
:star: PR built and available in a preview **peaqnetwork-apps-pr-6** [here](http://apps-pr6.ci.peaq.network)

**Comment by niazhussain** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **dorloechter**
You can assign the PR to them by writing `/assign @dorloechter` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://jenkins-x.io/v3/develop/reference/chatops/?repo=peaqnetwork%2Fapps).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/peaqnetwork/apps/blob/dev/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by niazhussain** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **dorloechter**
You can assign the PR to them by writing `/assign @dorloechter` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://jenkins-x.io/v3/develop/reference/chatops/?repo=peaqnetwork%2Fapps).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/peaqnetwork/apps/blob/dev/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by kamrangul2404** [comment]
approved

**Pull request merged**